sips/sip-012/sip-012-cost-limits-network-upgrade.md

```diff
@@ -0,0 +1,779 @@
+height at which a network-upgrade, described in detail in [this Github
+discussion](https://github.com/blockstack/stacks-blockchain/discussions/2845).
+Unfortunately, this path would take at least 4 months in the best-case scenario
+(factoring in time to test and deploy the voting contract, time to execute the
+vote, time for a miner veto window and time to clear the votes).
+
+This SIP posits that the ongoing network congestion warrants a more expedient
+route to change the cost-limits, one that does not rely on an on-chain voting
+contract.
+
+Equally of note, we consider this SIP as a method of last resort and
+that considering the circumstances an exception is justified. All future network
+upgrades should use the voting contract (if appropriate); all hard-forks must
+follow the processes described in SIP-000 and SIP-011.
+
+
+# Specification
+
+## Proposal
+
+In the text below, "Stacks 2.05" refers to the proposed network-upgrade for
+cost-limits.
+
+The SIP authors will propose a Bitcoin block number at which the new cost-limits
+take effect. The block number should be at least two calendar weeks from when
+this SIP transitions into “Accepted” state, so as to provide sufficient time for
+node operators to upgrade. Tentatively this block number would be chosen to fall
+on November 29th or November 30th, 2021.
+
+Miners, developers, Stackers and community members can demonstrate their support
+(or lack thereof) for this network upgrade by sending Bitcoin transactions:
+
+* There will be two Bitcoin addresses whose UTXOs will be used to tally the
+  vote: a "yes" address, and a "no" address. [TODO: add details on the specific
+  addresses]. To vote, you would send a small amount of BTC to either one of
+  these addresses.
+* The transaction's scriptSig must be signed by either the Stacker's PoX reward
+  address's public key(s), or the public key(s) of their Stacks address (the
+  option is provided here because not all Stackers have access to their PoX
+  addresses). In either case, the vote commits the Stacker's
+  most-recently-locked STX to "yes" or "no" if the Stacker had some STX locked
+  in the past two reward cycles at the time of the vote.
+
+Nodes that run Stacks 2.05 must put `0x05` in the memo field. Block-commit
+transactions that do not have `0x05` will be considered invalid. The purpose of
+this change is to ensure that in the unlikely event some miners didn't know
+about this SIP, they will quickly find out because their blocks will never be
+confirmed.
+
+The SIP will be considered Recommended if the vote to activate Stacks 2.05
+passes. This requires:
+
+* 2/3 of all votes passed are "yes", weighted by the STX they represent
+* At least 60 million STX are represented by the "yes" votes. This is 2x the
+  largest Stacker right now.
+
+The rationale for this voting procedure is that it simultaneously gives the
+community a way to veto the SIP while also accommodating a low-ish turnout. The
```

Comment:
So, my comment text wasn't meant to be copy/pasted directly ;)  "low-ish" isn't an appropriate vernacular for a standards-track document.  Just use "somewhat low" or the like.